client/sites-dashboard/components/sites-table.tsx

```diff
@@ -115,36 +130,39 @@ export function SitesTable( { className, sites, isLoading = false }: SitesTableP
 	}, [ masterbarHeight ] );
 
 	return (
-		<Table className={ className }>
-			<THead
-				blockOffset={ masterbarHeight }
-				ref={ headerRef }
-				style={ isHeaderStuck ? headerShadow : undefined }
-			>
-				<Row>
-					<th style={ { width: '50%' } }>{ __( 'Site' ) }</th>
-					<th style={ { width: '20%' } }>{ __( 'Plan' ) }</th>
-					<th>{ __( 'Status' ) }</th>
-					<th>{ __( 'Last Publish' ) }</th>
-					<th style={ { width: '24px' } }></th>
-				</Row>
-			</THead>
-			<tbody>
-				{ isLoading &&
-					Array( N_LOADING_ROWS )
-						.fill( null )
-						.map( ( _, i ) => (
-							<SitesTableRowLoading
-								key={ i }
-								columns={ 5 }
-								delayMS={ i * 150 }
-								logoProps={ { width: 108, height: 78 } }
-							/>
-						) ) }
-				{ sites.map( ( site ) => (
-					<SitesTableRow site={ site } key={ site.ID }></SitesTableRow>
-				) ) }
-			</tbody>
-		</Table>
+		<>
+			<Table className={ className }>
+				<THead
+					blockOffset={ masterbarHeight }
+					ref={ headerRef }
+					style={ isHeaderStuck ? headerShadow : undefined }
+				>
+					<Row>
+						<th style={ { width: '50%' } }>{ __( 'Site' ) }</th>
+						<th style={ { width: '20%' } }>{ __( 'Plan' ) }</th>
+						<th>{ __( 'Last Publish' ) }</th>
+						<th>{ __( 'Status' ) }</th>
+						<th style={ { width: '24px' } }></th>
```

Comment:
A typo looks like it's crept in somehow where the column headers have flipped the wrong way

<img width="294" alt="CleanShot 2022-10-07 at 16 20 03@2x" src="https://user-images.githubusercontent.com/1500769/194461043-480fb392-98e0-45aa-b20e-fd048ffc2d01.png">


```suggestion
						<th style={ { width: '20%' } }>{ __( 'Plan' ) }</th>
						<th>{ __( 'Status' ) }</th>
						<th>{ __( 'Last Publish' ) }</th>
						<th style={ { width: '24px' } }></th>
```